### DISCUSSION TITLE
[help]  scroll feature

### DISCUSSION THREAD

**Discussion by huntye1** [gilbarbara/react-joyride#683] [Q&A] [answered]
hello, first,  thanks this wonderful library.
but i have one problem.

demo like: https://codesandbox.io/s/festive-frog-oxgnl?file=/src/App.js

i know the beacon acting like  sticky is the feature. but my need is that beacon and tooltip follow the target.

i have read docs carefully. but no api found for this case. 

if there is any apis do work, plz tell me. or is  this  not support now ?

**Comment by huntye1** [comment]
some api like custom relative container?

**Comment by gilbarbara** [comment, marked as answer]
The beacon and tooltip follow the element in normal page scroll but it doesn't work with custom scroll elements.

**Reply by huntye1** [reply]
thanks for your reply. i think i can use control mode to get it.

**Comment by shanuflash** [comment]
@huntye1 have you figured out how this can be done?
